src/V3AstNodeExpr.h

```diff
@@ -975,6 +975,18 @@ class AstConst final : public AstNodeExpr {
         , m_num(this, 1, on) {
         dtypeSetBit();
     }
+    class All0 {};
+    AstConst(FileLine* fl, All0)
+        : ASTGEN_SUPER_Const(fl)
+        , m_num(this, "'0") {
+        initWithNumber();
+    }
```

Comment:
I don't see this suppressing the error check.  You did seem to test the supply constraints with the new flag, so I'm unsure how this code is passing without disabling the warning here.

```suggestion
        , m_num(this, "'0") {
        initWithNumber();
        nodep->fileline(new FileLine{nodep->fileline()});
        nodep->fileline()->warnOff(V3ErrorCode::NEWERSTD, true);
    }
```

And similar below

Reply:
Thanks! I wasn't suppressing the warning because I didn't know how, so thanks for showing me! I just added the suppression